app/SearchApi/SearchApi.Web/Notifications/WebHookNotifierMatchFound.cs

```diff
@@ -12,35 +12,33 @@
 
 namespace SearchApi.Web.Notifications
 {
-    public class WebHookNotifier : ISearchApiNotifier
+    public class WebHookNotifierMatchFound : BaseApiNotifier, ISearchApiNotifier<MatchFound> 
     {
 
         private readonly HttpClient _httpClient;
         private readonly SearchApiOptions _searchApiOptions;
-        private readonly ILogger<WebHookNotifier> _logger;
+        private readonly ILogger<WebHookNotifierMatchFound> _logger;
 
-        public WebHookNotifier(HttpClient httpClient, IOptions<SearchApiOptions> searchApiOptions,
-            ILogger<WebHookNotifier> logger)
+        public WebHookNotifierMatchFound(HttpClient httpClient, IOptions<SearchApiOptions> searchApiOptions,
+            ILogger<WebHookNotifierMatchFound> logger)
         {
             _httpClient = httpClient;
             _logger = logger;
             _searchApiOptions = searchApiOptions.Value;
         }
-                                                                                                                                                                                                                                            
-        public async Task NotifyMatchFoundAsync(Guid searchRequestId, MatchFound matchFound,
-            CancellationToken cancellationToken)
-        {
 
-            foreach (var webHook in _searchApiOptions.WebHooks)
+        public async Task NotifyEventAsync(Guid searchRequestId, MatchFound matchFound,
+           CancellationToken cancellationToken)
+        {
+            foreach (var webHook in _searchApiOptions.WebHooks.FindAll(x => x.EventName.Contains(nameof(MatchFound))))
```

Comment:
I would go with a strict comparison, ignoring the case, there could be some side effects witht the contains